/**:
  ros__parameters:
    NAMESPACE: /opp_racecar
    OPPONENT_NAMESPACE: /ego_racecar
    GLOBAL_PATH_TOPIC: /opp_racecar/waypoints
    DRIVE_TOPIC: /opp_drive
    COLOR: blue

    IS_VECTOR_COST: False
    STEERING_INCREMENT: 0.6109
    ACCELERATION_INCREMENT: 0.25
    ACTION_LST: [-1.0, -1.0, 
                 -1.0, 0.0,
                 -1.0, 1.0, 
                  0.0, -1.0,
                  0.0, 0.0,
                  0.0, 1.0,
                  1.0, -1.0, 
                  1.0, 0.0,
                  1.0, 1.0, 
                  1.0, -0.5,
                  1.0, 0.5]

    MAX_SPEED: 0.4  # m/s

    # e7_floor5
    # ACTION_INTERVAL: 0.25 # seconds
    # RECEDING_HORIZON: 1.5 # seconds
    # DT: 0.25

    # hallway - scalar parameters
    PROGRESS_RESET_RATIO: 0.75
    MAX_SPEED: 0.4  # m/s
    ACTION_INTERVAL: 0.1 # seconds
    RECEDING_HORIZON: 1.5 # seconds
    DT: 0.25

   # hallway - vector parameters

    # PROGRESS_RESET_RATIO: 0.75
    # MAX_SPEED: 0.4  # m/s
    # ACTION_INTERVAL: 0.1 # seconds
    # RECEDING_HORIZON: 3.0 # seconds
    # DT: 0.5

    # size for calculations, radial (width) and frontal (length) axes
    LR: 1
    LF: 1

    # how close bike center points are in pixels to count as collision
    COLLISION_RADIUS: 2.0
    PROXIMITY_SPREAD: 1.0
    BOUNDS_SPREAD: 3.0

    PROGRESS_WEIGHT: 1.0
    BOUNDS_WEIGHT: 4.0
    PROXIMITY_WEIGHT: 1.0

    # Should not be used at all, opponent is not vector cost
    OPP_COLLISION_RADIUS: 0.0
    OPP_PROXIMITY_SPREAD: 0.0
    OPP_BOUNDS_SPREAD: 0.0

    OPP_PROGRESS_WEIGHT: 0.0
    OPP_BOUNDS_WEIGHT: 0.0
    OPP_PROXIMITY_WEIGHT: 0.0